### PR TITLE
Added support for default token attributes

### PR DESCRIPTION
**Pull request by hbs**

### CODE REVIEW

File: warp10/src/main/java/io/warp10/continuum/Tokens.java [hunk 1]
```diff
@@ -299,55 +353,63 @@ public static ReadToken extractReadToken(String token) throws WarpScriptExceptio
...
+      }
     }
-    
-    for (String attr: blockedAttributes) {
-      if (attributes.containsKey(attr)) {
-        throw new RuntimeException("Invalid token attribute.");
+
+    if (null != defaults) {
+      if (null == attributes) {
+        attributes = new LinkedHashMap<String,String>(defaults);
```
Comment (steven-gueguen) [inline warp10/src/main/java/io/warp10/continuum/Tokens.java:376]: Defaults attributes wont' be propagated

**Review by steven-gueguen — CHANGES_REQUESTED** [review]

**Comment by hbs** [comment]
PTAL

File: etc/conf.templates/distributed/00-warp.conf.template [hunk 2]
```diff
@@ -103,6 +103,18 @@ warp.components =
...
+// be overriden if they exist
+//
+warp.token.read.attributes.default = { '.cap:limits' '' }
+
+//
+// WarpScript map defining the default attributes to allocate to write tokens. Existing attributes will not
+// be overriden if they exist
+//
+#warp.token.write.attributes.default = { '.nodelete' '' }
+
```
Comment (pi-r-p) [inline etc/conf.templates/distributed/00-warp.conf.template:117]: Not the best example, .nodelete leads to a non delete instance.

**Review by pi-r-p — CHANGES_REQUESTED** [review]

**Comment by hbs** [comment]
PTAL

**Review by steven-gueguen — APPROVED** [review]

**Pull request merged**